test_windows.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# Run CNI plugin tests.
+#
+set -e
+
+source ./build_windows.sh
+
+echo "Running tests"
+
+PLUGINS=$(cat plugins/windows_only.txt | tr '\n' ' ')
+GINKGO_FLAGS="-p -r --randomizeAllSpecs --randomizeSuites --failOnPending --progress pkg/hns $PLUGINS"
```

Comment:
I'm not thrilled with this explicit `pkg/hns` but I'm also not ready to say we need a new abstraction here either.  If we see other windows-only packages that aren't getting run in travis because of omission from this list, then we can revisit.